website/content/wgs/operator/charter/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Operator Working Group Charter
+description: The charter describes the mission and tactics of the Operator working group (WG)
+---
+
+# Operator Working Group Charter
+
+## Interested Parties
```

Comment:
We've been removing these lists of names because a) they're out of date - many of these parties are no longer interested and b) it misleads newcomers to thinking they need to submit PRs to add themselves.

My vote would be to remove the "Interested Parties" section here, particularly since this WG is a bit of a placeholder in any case.